SalesforceCommerceCloud/pwa-kit
# [V2] allow custom webpack config in projects

New PR following up on #378 and #401

# Description

This PR modifies the `mrt build` script and `build-dev-server` to allow projects to use a custom webpack config.

To add a custom webpack config, projects should add a `webpack.config.js` file in the root of their project directory (In this monorepo, one such place would be the pwa directory)

To inherit the default webpack config in your custom webpack config, insert the following lines:

```
const config = require('pwa-kit-build/configs/webpack/config')

module.exports = config
```
 
# Types of Changes




# Changes

- Add support for a custom webpack config

# How to Test-Drive This PR

1. For each template app (ie. the PWA or Hello World), add a console.log statement in that project's webpack.config.js
2. Start the app. Verify that the console.log statement appears in your terminal
3. Run `npm run build`. Verify that the console.log statement appears in your terminal

3. Delete the project's webpack.config.js
4. Start the app. Verify at everything still works as normal

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
